qiita_pet/templates/analysis_selected.html

```diff
@@ -0,0 +1,59 @@
+{% extends sitebase.html %}
+{% block head %}
+<script type="text/javascript">
+function remove_proc_data(pid, sid) {
+	if(confirm('Are you sure you want to remove all samples for this processed data?')) { 
+		$('#proc' + pid).remove();
+		$('#proc' + pid + '-samples').remove();
+		// remove study if all proc data removed
+		if($('#study'+ sid + '-table tbody').children().length === 1) { $('#study'+sid).remove(); }
+	}
+}
+
+function remove_sample(sid, pid, sample) {
+		document.getElementById(pid + '@' + sample).remove();
+		// remove proc data if all samples removed
+		if($('#proc' + pid + '-samples-table tbody').children().length === 0) { $('#proc'+pid).remove(); $('#proc' + pid + '-samples').remove(); }
+		// remove study if all proc data removed
+		if($('#study'+ sid + '-table tbody').children().length === 1) { $('#study'+sid).remove(); }
+}
+
+</script>
+{% end %}
+
+{% block content %}
+{% from qiita_db.study import Study %}
+{% from future.utils import viewitems %}
+<h1>Selected Samples</h1>
+{% for sid, proc_datas in viewitems(sel_data) %}
+  {% set study = Study(sid) %}
+<div class="row" id="study{{sid}}">
+	<div class="col-md-12" style="border:2px solid #a1a1a1; border-radius: 15px;">
+	  <h2><a href="/study/description/{{sid}}">{{study.title}}</a></h2>
+	  <table class='table table-striped' id='study{{sid}}-table'>
+	  <tr>
+	  	<th></th><th>id</th><th>Datatype</th><th>Processed Date</th><th>Algorithm</th><th>Reference</th><th></th>
```

Comment:
Sorry to disagree about your previous statement. What about this layout?
![screen shot 2015-04-03 at 10 50 01 am](https://cloud.githubusercontent.com/assets/2501478/6986132/3579854e-d9ef-11e4-86df-72af9f7f4eec.png)